terraform/shared/providers.tf

```diff
@@ -1,6 +1,5 @@
 provider "aws" {
-  region  = var.region
-  version = "~> 3"
```

Comment:
are the versions we deployed with in a lock file or something similar? the underlying eks module the community provides hasn't been the best in terms of upgrades. We added these versions in an effort to get a useful error at least when they decide to just randomly do provider upgrades.

Reply:
Yep, it makes sense.
I removed it because when I was upgrading the terraform, I received the warning `Version constraints inside provider configuration blocks are deprecated`, but I should include it in the `state.tf` file.
Just included.

Reply:
ahh, just moving locations.